_includes/home/calendar.html

```diff
@@ -0,0 +1,110 @@
+<section class="calendar" id="calendar">
+
+<div class="container">
+    <div class="row">
+        <div class="col-lg-12 text-center">
+
+            <div class="intro-text">
+                <h2 class="section-heading">TeachOSM Calendar</h2>
+            </div>
+
+            <br>
+
+            <!--<div id="content">
+              <div class="titlebox">Events</div>
+                <ul id="legend-list">
+                    <li><dl>
+                            <dt>class 1:</dt>
+                            <dd> <div class="legend_color" id="class1-color"></div> </dd>
+                        </dl></li>
+                    <li><dl>
+                            <dt>class 2:</dt>
+                            <dd> <div class="legend_color" id="class2-color"></div> </dd>
+                        </dl></li>
+                    <li><dl>
+                            <dt>class 3:</dt>
+                            <dd> <div class="legend_color" id="class3-color"></div> </dd>
+                        </dl></li>
+                </ul>
+            </div>-->
+
+            <br>
+
+            <div id='insert-calendar'></div>
+        
+        </div>
+    </div>
+</div>
+
+
+<!--   http://www.mikesmithdev.com/blog/jquery-full-calendar-with-ui-modal-dialog/   -->
+<div id="eventContent" title="Event Details" style="display: none;">
+    Start: <span id="startTime"></span><br>
+    End: <span id="endTime"></span><br><br>
+    <div id="eventInfo"></div>
+    </br>
+    <p><strong><a id="eventLink" target="_blank">Read More</a></strong></p>
+</div>
+
+</section>
+
+<script>
+$(document).ready(function() {
```

Comment:
Could you move this js to a new file in `assets/js`? I think that would be the best place for it. I could also see a new top-level directory being useful for javascript that we write ourselves.

Reply:
ok, creating subfolder named 'custom' in js folder.